.github/workflows/build.yaml

```diff
@@ -208,8 +204,7 @@ jobs:
       - uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          # FIXME: replace with stable ASAP
-          toolchain: nightly-2022-07-22
+          toolchain: stable
```

Comment:
This should be the same "stable = fail, beta = warn" scenario, though nightly won't be required since docs.rs won't be using it.